syntax = "proto3";
package org.kobe.xbot.Utilities.Entities;
message Array {
    repeated Value values = 1;

    message Value {
        oneof value {
            string string_value = 1;
            int32 int_value = 2;
            float float_value = 3;
            double double_value = 4;
            bool bool_value = 5;
            int64 int64_value = 6;
            uint32 uint32_value = 7;
            uint64 uint64_value = 8;
            bytes bytes_value = 9;
        }
    }
}
